Script/AtomicEditor/editor/Preferences.ts

```diff
@@ -32,7 +32,7 @@ class Preferences {
 
     updateRecentProjects(write: boolean = false): void {
 
-        for (var i in this._prefs.recentProjects) {
+        for (var i = 0; i < this._prefs.recentProjects.length; i++) {
             var path = this._prefs.recentProjects[i];
```

Comment:
this._prefs.recentProjects is a ```string[]```  not an object, so you will want to iterate over it like any other array, and not like a key/value object.  TS 1.8 flagged this as an error.